main.py

```diff
@@ -15,102 +15,100 @@
 SUPPORT_SERVER = "https://discord.gg/CgH6Sj6"
 INVITE = "https://discord.com/api/oauth2/authorize?client_id=715545167649570977&permissions=980675863&scope=bot"
 
-client = commands.Bot(commands.when_mentioned_or(PREFIX), case_insensitivity=True, owner_id=688275913535914014)
-
-status = [
-    "😁Working At The Codin' Hole! Join me at https://discord.gg/aYF76yY",
-    "▶Check out My Creator's Youtube channel : https://www.youtube.com/channel/UC3S4lcSvaSIiT3uSRSi7uCQ/",
-    f"Ping me using {PREFIX}help",
-    "Official Instagram of My Creator ❌ https://instagram.com/the.codin.hole/",
-    "Ready To Work and Get Worked! My Github 🔆 https://github.com/janaSunrise",
+extensions = [
+    "cogs.codesandbox",
+    "cogs.commands",
+    "cogs.converters",
+    "cogs.custom",
+    "cogs.emotes",
+    "cogs.events",
+    "cogs.fun",
+    "cogs.games",
+    "cogs.infog",
+    "cogs.moderation",
+    "cogs.study",
+    "cogs.sudo",
+    "cogs.support",
+    "cogs.tools",
 ]
+# "cogs.coding"
+
+
+class Bot(commands.Bot):
+    def __init__(self, *args, **kwargs) -> Bot:
```

Comment:
```suggestion
    def __init__(self, *args, **kwargs) -> None:
```
This doesn't seem to return a `Bot` instance, the type hint should be `None`

Reply:
Did you check if the typehint accepts subclasses ? Or is Travis really that annoying ?

Reply:
These checks aren't a feature of travis-ci, but of our pre-commit which uses `flake8-annotations`.  Those checks doesn't explicitly fail, because there is type defined, but it's a wrong type so it needs to be fixed.

Even if type hinting accepted subclasses, I want to use this new `Bot` class there in case there were some more functions added, etc. so that editors can detect this. If you want to read more about the reasoning for the type highlighting read through [**`CONTRIBUTING.md`**](https://github.com/The-Codin-Hole/HotWired-Bot/blob/6e776c3b99088e27767ae2bafaa1f41d9bc93a23/CONTRIBUTING.md) Type Hinting has a separate section there.